frontend/institution/institutionHeader.component.js

```diff
@@ -42,18 +42,18 @@
 
             /** Return the title of page according current state.
              */
-            instHeaderCtrl.getTitle = function getTitle(){
-                const getLimitedName = instHeaderCtrl.actionsButtons && 
-                    instHeaderCtrl.actionsButtons.getLimitedName(110);
-                const tileState = {
-                    [STATES.INST_TIMELINE]: getLimitedName,
-                    [STATES.INST_REGISTRATION_DATA]: "Dados cadastrais",
-                    [STATES.INST_LINKS]: "Vínculos Institucionais",
-                    [STATES.INST_DESCRIPTION]: "Descrição",
-                    [STATES.INST_MEMBERS]: "Membros",
-                    [STATES.INST_FOLLOWERS]: "Seguidores"
+            instHeaderCtrl.getTitle = function getTitle(){                
+                const instName = instHeaderCtrl.institution ? instHeaderCtrl.institution.name : "";
+                const limitedInstName = Utils.limitString(instName, 110);
+                
+                switch($state.current.name) {
+                    case STATES.INST_TIMELINE: 
```

Comment:
empty?

Reply:
If its not necessary, remove it

Reply:
Both states return the same thing, so there's no need to repeat the code